src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -74,9 +90,11 @@ private static void Main(string[] args)
         // Issues a GET request to the Bridge to determine whether it is alive.
         // A return of 'true' means the Bridge is healthy.  A return of 'false'
         // indicates the Bridge is not healthy, and 'errorMessage' describes the problem.
-        private static bool PingBridge(string host, int port, out string errorMessage)
+        // A healthy response will also populate properties with the current Bridge configuration.
+        private static bool PingBridge(string host, int port, out string errorMessage, out Dictionary<string, string> properties)
```

Comment:
Note: good comments but the level of review on this PR exceeds my ability to respond and still make progress on ToF.  I am going to debug what I just pushed up and come back to consider these later.  Whew, we need this level of feedback on product code PR's :smile: